bookwyrm/models/book.py

```diff
@@ -448,15 +449,98 @@ def to_edition_list(self, **kwargs):
 ]
 
 
+def validate_isbn10(maybe_isbn: str) -> None:
+    """Check if isbn10 mathes some expectations"""
+
+    if not (normalized_isbn := normalize_isbn(maybe_isbn)):
+        raise ValidationError(
+            _("%(value)s doesn't look like isbn"), params={"value": maybe_isbn}
```

Comment:
I think this should be phrased ever so slightly differently:

`_("%(value)s doesn't look like an ISBN"), params={"value": maybe_isbn}`